##Adopters

#Phenoscape
The Phenoscape projects collects and analyzes phenotypic character descriptions from a range of vertebrate species, using Uberon and PATO to describe evolutionary character states.

The Phenoscape project is both a major driver of and contributor to Uberon, contibuting thousands of terms. The teleost (bony fishes) component of Uberon was derived from the Teleost Anatomy Ontology, developed by the Phenoscape group. Most of the high level design of the skeletal system comes from the Vertebrate Skeletal Anatomy Ontology (VSAO), also created by the Phenoscape group. Phenoscape curators continue to extend the ontology, covering a wide variety of tetrapod structures, with an emphasis on the appendicular system.

See the [Phenoscape website](https://phenoscape.org/) for more details

#Bgee
Bgee is a database to retrieve and compare gene expression patterns between animal species. Bgee in using Uberon to annotate the site of expression, and Bgee curators one the major contributors to the ontology.

[BGee](https://bgee.org/)

#Gene Ontology
The Gene Ontology uses Uberon to classify developmental processes, and to provide additional contextual information on annotations, such as the location of a biological process.

See the [GO website](http://geneontology.org/)

#Monarch Initiative
Model systems are the cornerstone of biomedical research to investigate biological processes, test gene-based disease hypotheses, and develop and test disease treatments. The vast knowledge that we have about model systems can be better utilized if semantically aggregated and made queryable based on any number of facets, such as phenotypic similarity, network analysis, gene expression and function, and genomics. The Monarch Initiative aims to provide easy-to-use tools to navigate this data landscape, services for other resources, and educational outreach regarding the production of structured data for biomedical discovery.

The user of Uberon to bridge between different species is key to the phenotype mapping component of this project.

See the [Monarch Initiative website](https://monarchinitiative.org/) for more details

#EBI
The [EBI Samples Phenotypes and Ontology Team](https://www.ebi.ac.uk/about/spot-team) is using Uberon for describing biological samples and phenotypes. Recently, Uberon was integrated with the EFO (Experimental Factory Ontology), developed by the SPOT group.

#Global Alliance for Genomes and Health
The [Global Alliance for Genomes and Health](https://www.ga4gh.org/) is an international coalition, dedicated to improving human health by maximizing the potential of genomic medicine through effective and responsible data sharing.

The Global Alliance Data Working Group has proposed Uberon and the CL as a standard for the description of biological samples. See the metadata schema:

#eagle i
The eagle i project uses Uberon anatomical structures to collect information about cell lines and biospecimens. Since eagle-i is often concerned with non-model organism resources, Uberon + Taxonomy is the ideal mechanism to assert the source anatomical structure for these resources.

#FANTOM5
FANTOM is an international research consortium established by Dr. Hayashizaki and his colleagues in 2000 to assign functional annotations to the full-length cDNAs that were collected during the Mouse Encyclopedia Project at RIKEN. FANTOM has since developed and expanded over time to encompass the fields of transcriptome analysis. The object of the project is moving steadily up the layers in the system of life, progressing thus from an understanding of the ‘elements’ - the transcripts - to an understanding of the ‘system’ - the transcriptional regulatory network, in other words the ‘system’ of an individual life form.

FANTOM5 is using Uberon and CL to annotate samples allowing for transcriptome analyses with cell-type and tissue-level specificity.

See [FANTOM website](https://fantom.gsc.riken.jp/)

#ENCODE
The National Human Genome Research Institute (NHGRI) launched a public research consortium named ENCODE, the Encyclopedia Of DNA Elements, in September 2003, to carry out a project to identify all functional elements in the human genome sequence.

The [ENCODE](https://www.encodeproject.org/) Data Collection Center (DCC) uses a core set of ontologies:
- UBERON
- CL
- EFO
- OBI
- ChEBI
- SO
- GO
- See ENCODE DCC Ontologies

Malladi, V. S., Erickson, D. T., Podduturi, N. R., Rowe, L. D., Chan, E. T., Davidson, J. M., … Hong, E. L. (2015). Ontology application and use at the ENCODE DCC. Database : The Journal of Biological Databases and Curation, 2015, bav010–. [doi:10.1093/database/bav010](https://academic.oup.com/database/article/doi/10.1093/database/bav010/2433142)

Sloan, C. A., Chan, E. T., Davidson, J. M., Malladi, V. S., Strattan, J. S., Hitz, B. C., … Cherry, J. M. (2015). ENCODE data at the ENCODE portal. Nucleic Acids Research, gkv1160–. [doi:10.1093/nar/gkv1160](https://academic.oup.com/nar/article/44/D1/D726/2502655)

#neXtProt
[neXtProt](http://www.nextprot.org/) is an on-line knowledge platform on human proteins. It strives to be a comprehensive resource that provides a variety of types of information on human proteins, such as their function, subcellular location, expression, interactions and role in diseases.

[neXtProt](http://www.nextprot.org/) is using Uberon as the main vaculary for describing site of protein localization in animals.

#FEED
The NSF FEED (Feeding Experiments End-user Group) working group is developing a database of mammalian feeding muscle data. The FEED developers are responsible for the craniofacial muscle aspects of Uberon.

#SciCrunch
Uberon is the anatomical ontology used as part of the SciCrunch integrated search system and dkNET projects.

#CELLPEDIA
See Hatano et al [in Database](https://academic.oup.com/database/article/doi/10.1093/database/bar046/468477)

#International Human Epigenomics Consortium
The International Human Epigenome Consortium (IHEC) is a global consortium with the primary goal of providing free access to high-resolution reference human epigenome maps for normal and disease cell types to the research community. IHEC is working to define standards for epigenomic mapping and metadata. Uberon and CL are the IHEC standard ontologies for tissue and cell line names.

For more details, see the [IHEC Standards page](http://ihec-epigenomes.org/research/reference-epigenome-standards/).

#NIH LINCS
The NIH Library of Integrated Network-based Cellular Signatures (LINCS) project aims to create a network-based understanding of biology by cataloging changes in gene expression and other cellular processes that occur when cells are exposed to a variety of perturbing agents, and by using computational tools to integrate this diverse information into a comprehensive view of normal and disease states that can be applied for the development of new biomarkers and therapeutics.

This project recommends the usage of Uberon and the OBO Cell Ontology for describing organs, tissues and cells for annotating reagents and assays.

See [Vempati et al](https://journals.sagepub.com/doi/full/10.1177/1087057114522514)

#The Alexandria Archive Institute
In September, 2012, the AAI launched [Exploring Biogeography of Early Domestic Animals using Linked Open Data](https://alexandriaarchive.org/linked-data/), a one-year project using Linked Open Data to enhance archaeological data sets. Skeletal element data are also linked using Uberon.

See:

[linked data](http://alexandriaarchive.org/projects/linked-data/)
[Mixing Models for Communicating Research Data in Archaeology, Kansa et al, International Journal for Digital Curation,Vol 9, No. 1 (2014)](http://alexandriaarchive.org/wp-content/uploads/2012/11/idcc14-Kansa-Kansa-Arbuckle-researchpaper-final.pdf)